tests/unit/test_mxnet.py

```diff
@@ -372,3 +373,25 @@ def test_attach_custom_image(sagemaker_session):
     estimator = MXNet.attach(training_job_name='neo', sagemaker_session=sagemaker_session)
     assert estimator.image_name == training_image
     assert estimator.train_image() == training_image
+
+
+def test_estimator_script_mode_launch_parameter_server(sagemaker_session):
+    mx = MXNet(entry_point=SCRIPT_PATH, role=ROLE, sagemaker_session=sagemaker_session,
+               train_instance_count=INSTANCE_COUNT, train_instance_type=INSTANCE_TYPE,
+               distributions=LAUNCH_PS_DISTRIBUTIONS_DICT, framework_version='1.3.0')
```

Comment:
the default is being left at 1.2.1 so that this isn't a breaking change. Also the point of these three new unit tests is to be deliberate about the framework version

Reply:
I guess what i was thinking was do we need a latest_version constant. Are we going to change this framework version number here.

Reply:
this test just needs a version >= 1.3.0; no need to change it with later versions